lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -44,18 +44,32 @@ def test_output_against_profile(klass,
 
         lines_to_validate_parameter = get_lines_to_validate(bulk_lines_to_validate)
 
-        file = output.find { |item| item['type'] == klass }
+        file_list = output.find_all { |item| item['type'] == klass }
 
-        skip "Bulk Data Server export does not have #{klass} data" if file.nil?
+        if file_list.empty?
+          pass 'No Medication resource in output.' if klass == 'Medication'
+
+          skip "Bulk Data Server export does not have #{klass} data"
+        end
 
         validate_all = lines_to_validate_parameter[:validate_all]
         lines_to_validate = lines_to_validate_parameter[:lines_to_validate]
 
+        @patient_ids_seen = Set.new if klass == 'Patient'
+
         omit 'Validate has been omitted because line_to_validate is 0' if !validate_all && lines_to_validate.zero? && klass != 'Patient'
 
-        success_count = check_file_request(file, klass, validate_all, lines_to_validate, must_supports)
+        success_count = 0
+
+        file_list.each do |file|
+          success_count += check_file_request(file, klass, validate_all, lines_to_validate, must_supports)
+        end
+
+        if success_count.zero? && (validate_all || lines_to_validate.positive?)
+          pass 'No Medication resource in output.' if klass == 'Medication'
 
-        skip "Bulk Data Server export for #{klass} is empty" if success_count.zero? && (validate_all || lines_to_validate.positive?)
+          skip "Bulk Data Server export for #{klass} is empty"
```

Comment:
```suggestion
          skip "Bulk Data Server export did not provide any #{klass} resources."
```

Reply:
oh shoot something weird happened.  The point here is that we should use `omit` not skip if medication.

Reply:
fixed